docs/API.md

```diff
@@ -480,31 +480,42 @@ A single glob or array of globs that indicate which files to watch for changes.
 #### opts
 Type: `Object`
 
-Options, that are passed to [`gaze`][gaze].
+Options, that are passed to [`chokidar`][chokidar].
 
 #### fn
 Type: `Function`
 
 An [async](#async-support) function to run when a file changes.
 
-`gulp.watch` returns an `EventEmitter` object which emits `change` events with
-the [gaze] `event`:
+`gulp.watch` returns a wrapped [chokidar] FSWatcher object. If provided,
```

Comment:
needs to pull in important (ie close) docs from the chokidar repo not just link to it - we have historically had issues just linking to other repos for docs - it makes discoverability hard